### ISSUE TITLE
SMTP Forward or Proxy fails after authentification

### ISSUE THREAD

**Issue by PEM--** [haraka/Haraka#1369]
### Haraka version
2.8.0-apha4 & 2.7.3

### Expected behavior
Forward emails to a proxy/

### Observed behavior
Crash with an unexpected usage of `address`:
```txt
[CRIT] [-] [core] TypeError: Object #<pluggableStream> has no method 'address'
[CRIT] [-] [core]     at pluggableStream.<anonymous> (/usr/lib/node_modules/Haraka/smtp_client.js:127:57)
[CRIT] [-] [core]     at pluggableStream.emit (events.js:98:17)
[CRIT] [-] [core]     at Socket.<anonymous> (/usr/lib/node_modules/Haraka/tls_socket.js:51:14)
[CRIT] [-] [core]     at Socket.emit (events.js:117:20)
[CRIT] [-] [core]     at Object.afterConnect [as oncomplete] (net.js:887:10)
```

### Steps to reproduce
Set the `smtp_proxy` or the `smtp_forward` plugin.

**Comment by baudehlo** [comment]
Already fixed by #1362

**Comment by PEM--** [comment]
Cool. Thanks :+1:
